samcli/commands/init/__init__.py

```diff
@@ -2,24 +2,36 @@
 """
 Init command to scaffold a project app from a template
 """
+import itertools
+import json
 import logging
+import os
+import subprocess
 
 import click
 
-from samcli.cli.main import pass_context, common_options
+from samcli.cli.main import pass_context, common_options, global_cfg
+from samcli.commands.exceptions import UserException
+from samcli.local.common.runtime_template import RUNTIMES, SUPPORTED_DEP_MANAGERS
 from samcli.lib.telemetry.metrics import track_command
-from samcli.local.common.runtime_template import INIT_RUNTIMES, SUPPORTED_DEP_MANAGERS, DEFAULT_RUNTIME
+from samcli.commands.init.init_generator import do_generate
```

Comment:
Some of these imports should be pushed down into the `do_cli` function. This is to keep `--help` text load time low, otherwise we will be loading all of these modules.

Reply:
Which imports?

Reply:
This seems to create other problems, we can discuss in private.